metadata-service/factories/src/main/resources/application.yml

```diff
@@ -10,6 +10,7 @@ authentication:
       configs:
         # Key used to validate incoming tokens. Should typically be the same as authentication.tokenService.signingKey
         signingKey: ${DATAHUB_TOKEN_SERVICE_SIGNING_KEY:WnEdIeTG/VVCLQqGwC/BAkqyY0k+H8NEAtWGejrBI94=}
+        saltingKey: ${DATAHUB_TOKEN_SERVICE_SALTING_KEY:WnEdIeTG/VVCLQqGwC/BAkqyY0k+H8NEAtWGejrBI93=}
```

Comment:
Minor nit: can we change this to something else. 

We are also likely going to have to include this in our helm charts as an auto-generated value.